sass/pages/restart-pass/_restart_pass.scss

```diff
@@ -0,0 +1,59 @@
+.restart-body{
+	background-color: #F7F7F7;
+	
+	.restart-section{
+		background-color: $white;
+		text-align: center;
+		
+		.restart-div{
+			width:25%;
+			@include center(true,true);
```

Comment:
Try not to mix mixins with the other properties, make it readable: mixins first, then a line break, then the rest of properties.